src/System.Security.Principal.Windows/ref/System.Security.Principal.Windows.cs

```diff
@@ -7,13 +7,11 @@
 
 namespace Microsoft.Win32.SafeHandles
 {
-    [System.Security.SecurityCriticalAttribute]
```

Comment:
Given that we support this contract on the full framework I believe we need to keep these SecurityCriticalAttributes in the contract. @ericstj what is your thoughts?